### PR TITLE
Add doughnut maker to UI

### PR DESCRIPTION
**Pull request by holygits**
Adds:
- Doughnut Maker UI page

Changes:
- Move webpack and required build deps e.g. `@babel/*` into this project's `devDependencies`- instead of `@polkdot/dev` vendored versions (gives some more flexibility on managing versions as wasm config/support is fragile in different webpack versions)

TODO:
- ~[ ] implement schnorrkel signing~ (parking for now)
- [x] fix e25519 signing
- [ ] Fix webpack build :/

### CODE REVIEW

File: packages/app-toolbox/src/Sign.tsx [hunk 1]
```diff
@@ -92,13 +92,21 @@ function Sign ({ className }: Props): React.ReactElement<Props> {
...
         .then(({ signature }): void => setSignature(signature));
     } else {
-      setSignature(u8aToHex(
-        currentPair.sign(
-          isHexData
-            ? hexToU8a(data)
-            : stringToU8a(data)
-        )
-      ));
+      console.log(currentPair);
```
Comment (holygits) [inline packages/app-toolbox/src/Sign.tsx:95]: debugging, will remove

File: packages/app-doughnut/src/index.tsx [hunk 2]
```diff
@@ -0,0 +1,155 @@
+// Copyright 2017-2020 @polkadot/app-123code authors & contributors
```
Comment (KarishmaBothara) [inline packages/app-doughnut/src/index.tsx:1]: // Copyright 2017-2020 @polkadot/app-doughnut authors & contributors

File: packages/apps-routing/src/doughnut.ts [hunk 3]
```diff
@@ -0,0 +1,25 @@
+// Copyright 2017-2020 @polkadot/apps-routing authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+import { Routes } from './types';
+
+import Template from '@cennznet/app-doughnut';
```
Comment (KarishmaBothara) [inline packages/apps-routing/src/doughnut.ts:7]: import **Doughnut**

File: packages/app-doughnut/README.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
+# @polkadot/app-123code
```
Comment (KarishmaBothara) [inline packages/app-doughnut/README.md:1]: may need to update readme

**Pull request merged**